core/src/main/java/io/substrait/relation/ProtoRelConverter.java

```diff
@@ -103,6 +102,12 @@ public Rel from(io.substrait.proto.Rel rel) {
       case HASH_JOIN -> {
         return newHashJoin(rel.getHashJoin());
       }
+      case NESTED_LOOP_JOIN -> {
```

Comment:
Why is the order Hash, Nested, Merge over alphabetical sort?

Reply:
This is the order in the Substrait spec, but I agree alphabetical would be better. Will update!

Reply:
And here.

Reply:
oof, thank you